src/admin-views/events-meta-box.php

```diff
@@ -233,34 +234,25 @@ class='alignleft'
 							class="tribe-dropdown"
 							data-prevent-clear
 						>
-							<?php
-							if ( isset( $_EventCurrencyPosition ) && 'suffix' === $_EventCurrencyPosition ) {
-								$suffix = true;
-							} elseif ( isset( $_EventCurrencyPosition ) && 'prefix' === $_EventCurrencyPosition ) {
-								$suffix = false;
-							} elseif ( true === tribe_get_option( 'reverseCurrencyPosition', false ) ) {
-								$suffix = true;
-							} else {
-								$suffix = false;
-							}
+							<?php // phpcs:ignore -- Legacy naming.
+							$suffix = $_EventCurrencyPosition ? ( 'suffix' === $_EventCurrencyPosition ? true : false ) : ( tribe_get_option( 'reverseCurrencyPosition' ) ?: false );
 							?>
 							<option value="prefix"> <?php _ex( 'Before cost', 'Currency symbol position', 'the-events-calendar' ) ?> </option>
-							<option value="suffix"<?php if ( $suffix ) {
-								echo ' selected="selected"';
-							} ?>><?php _ex( 'After cost', 'Currency symbol position', 'the-events-calendar' ) ?></option>
+							<option value="suffix" <?php selected( $suffix, true ); ?>><?php _ex( 'After cost', 'Currency symbol position', 'the-events-calendar' ) ?></option>
```

Comment:
It's worth pointing out that the `_e()` functions [don't escape](https://developer.wordpress.org/reference/functions/_e/#source). While the chance of using this as an attack vector is small, it's probably best practice to do `echo esc_html( _x() );`

Reply:
FWIW WP VIP has dinged me on this kind of thing in the past.

Reply:
@Camwyn My change here was to remove this

```
<?php if ( $suffix ) { 
   echo ' selected="selected"';
} ?>>
```
and change to this

```
<?php selected( $suffix, true ); ?>
```
Because I couldn't stand a three-liner as an html element's attribute.

The `_ex()` was already there. I will escape it, no problem.